CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Change Log
 
+## Version 1.31.1
+* Release date: July 29, 2021
+* Release status: General Availability
+
+## Hotfix Release
+
+- Fix for [#16436 Database Connection Toolbar Missing](https://github.com/microsoft/azuredatastudio/issues/16436)
+* New Notebook Features:
```

Comment:
We should not repeat points from the main release. If you look at previous hotfixes, we only include the bug fixes.

Reply:
Changes done as mentioned. Please check.